src/Storage/Device/BackBlaze.php

```diff
@@ -2,9 +2,10 @@
 
 namespace Utopia\Storage\Device;
 
-use Utopia\Storage\Device\S3;
+use Utopia\Storage\Device\Generic;
```

Comment:
We should have a better name for `Generic`?
Can we use `S3` as base instead of creating new `Generic`?

Reply:
S3 is also one of Generic child's.
Can be S3Base? or S3Compatible

Reply:
What's the difference in S3 itself and Generic? if there is a difference then yes Generic can `S3Base` - `S3Base` sounds more reasonable than `S3Compatible`.

Reply:
ok